logbook-spring-boot-autoconfigure/pom.xml

```diff
@@ -48,8 +48,8 @@
             <artifactId>logbook-servlet</artifactId>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
One thing I missed though, I think you need to add the `org.springframework.boot.autoconfigure.AutoConfiguration.imports` file with `org.zalando.logbook.autoconfigure.LogbookAutoConfiguration` in it for autoconfiguration to work. As per [this comment in another PR](https://github.com/zalando/logbook/pull/1375#discussion_r1027269042)

Reply:
Thanks, fixed: https://github.com/zalando/logbook/pull/1417/commits/a39e77c964f75f0224801d1b19e23ac2e3e76e75